pom.xml

```diff
@@ -464,6 +466,8 @@
     <version.org.jacoco>0.6.3.201306030806</version.org.jacoco>
 -->
     <version.org.sonatype.plugins.nexus-staging-maven-plugin>1.6.6</version.org.sonatype.plugins.nexus-staging-maven-plugin>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
```

Comment:
@tomjenkinson just wanted to point out that I've added this here. Is there a particular reason why we had a lower version before? Adding 1.8 allows us using lambdas, streams and other stuff in a new code.